### ISSUE TITLE
RNNT loss is constantly zero

### ISSUE THREAD

**Issue by Rajratnpranesh** [HawkAaron/warp-transducer#92]
Hey

I am trying to train the model but the loss is constant = '0'. I found that the loss output from the PyTorch binding of WarpRNNT is a zero tensor. I am getting an error of 'src/binding.cpp:151: unsupported data type'. Is this is the reason why WarpRNNT is not able to calculate loss (I charged the datatype from float16 to 32 as sum() operation was not possible with float16 then after summing back to float16). I appreciate any pointer.

Ty

**Comment by ccc0117** [comment]
I have same issue. Can someone help?

**Comment by gowtham1997** [comment]
any updates on this issue?
I was trying to use warprnnt with espnet and facing similar error when i use fp16 mode 

It trains well when I switch to fp32
